pom.xml

```diff
@@ -19,11 +19,13 @@
 
     <properties>
         <spring-data-jdbc.version>1.2.1.RELEASE</spring-data-jdbc.version>
-        <springfox-swagger.version>2.9.2</springfox-swagger.version>
+        <springfox-swagger.version>3.0.0</springfox-swagger.version>
         <jacoco.version>0.8.5</jacoco.version>
-
         <docker.jib-maven-plugin.version>1.3.0</docker.jib-maven-plugin.version>
         <docker.image.prefix>springcommunity</docker.image.prefix>
+        <jsr305.version>3.0.2</jsr305.version>
+        <jackson-databind-nullable.version>0.2.1</jackson-databind-nullable.version>
+        <mapstruct.version>1.4.1.Final</mapstruct.version>
```

Comment:
We could add a comment to precise that we could not use the version inherited from Spring Boot BOM due to the `maven-compiler-plugin` configuration.